test/suite/test_rollback_to_stable10.py

```diff
@@ -302,6 +302,12 @@ def prepare_range_updates(session, cursor, ds, value, nrows, prepare_config):
             done.set()
             ckpt.join()
 
+        # Check the history store file size before the simulated crash
```

Comment:
First, all comments must be complete sentences and end in a period. I'd suggest slightly rewording to:
Check that the history store file has been used and has non-zero size before the simulated crash.